src/js/index.js

```diff
@@ -2061,20 +2261,27 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
         item: {
           font: {
-            color: 'dark-3',
+            color: darkColors[6],
```

Comment:
Does dark-7 not work?

```suggestion
            color: 'dark-7',
```